features/dom.yml

```diff
@@ -0,0 +1,90 @@
+name: DOM
+description: TODO
+spec: https://dom.spec.whatwg.org/
+compat_features:
```

Comment:
This is missing some key things, namely `Document` and `Element`. I think the DOM spec does a nice job telling us what the DOM is in the [Introduction to "The DOM"](https://dom.spec.whatwg.org/#introduction-to-the-dom) section. Going based on that section, I figure that to represent an HTML document, we're going to need, at a minimum (plus pretty much everything else already here):

- `Document`
- `DocumentType`
- `Element`
- `Text`
- `CharacterData`

Reply:
Document was already in #1983 before this PR was created, but closing that to fold into this.